source/gdswiki/iterate-and-improve-frequently/iterate-and-improve-frequently.html.md.erb

```diff
@@ -14,15 +14,16 @@ Make sure you have the capacity, resources and technical flexibility to iterate
 ### Running your service in a sustainable way
 
 * User research was carried out to identify several key user groups and their key user needs, at both Discovery and Alpha stages of the project. These identified user needs were incorporated into the Beta product, and tested against user experiences with the product.
-* We initially launched the product in a private beta in January 2022. However, we continued to develop the product from this point onwards, adding key post-MVP features. This included API access, based on requests from satellite operators and orbital analysts for such a feature.
+* We initially launched the product in a private beta in January 2022. However, we have continued to develop the product from this point onwards, adding key post-MVP features. This included API access, based on requests from satellite operators and orbital analysts for such a feature.
 * The private beta gives the users the opportunity to test the service, identify any technical problems, highlight key areas for improvement and increase usability. The interface was also tested before private beta in a user testing event with 10 students from Cranfield University.
-* Overall we have made over 40 releases since public beta started
+* Since we began public beta we have made over 1000 deployments and continue to design, develop and deploy new features to improve the product.
```

Comment:
Can we change to say
> Since first commit, pipelines have run more than 1000 times to iterate and make amendments. We continue to design, develop and deploy new features to improve the product